README.md

```diff
@@ -194,7 +207,7 @@ presto --server localhost:8080 --catalog hive --schema default --user presto --f
 
 ## License
```

Comment:
I do not find an explicit **License** section in [vagrant-hashistack-readme](https://github.com/fredrikhgrelland/vagrant-hashistack/blob/master/README.md). Do we need it?